t/18-qemu.t

```diff
@@ -114,7 +114,7 @@ $proc  = qemu_proc('-foo', \%vars);
 @gcmdl = $proc->gen_cmdline();
 is_deeply(\@gcmdl, \@cmdl, 'Generate qemu command line for single existing UEFI disk using vars');
 
-@cmdl  = ([qw(create -f qcow2 -b), "$Bin/data/Core-7.2.iso", qw(raid/hd0-overlay0 11116544)]);
+@cmdl  = ([qw(create -f qcow2 -F qcow2 -b), "$Bin/data/Core-7.2.iso", qw(raid/hd0-overlay0 11116544)]);
```

Comment:
This is not right. The backing format for an ISO should be raw, not qcow2. If you look at the logs of a run with qemu 6.0.0 or earlier, you will see that it auto-detects raw, not qcow2, as the backing format. I already pointed this out in the ticket as the reason why I did not send this trivial PR myself.

Reply:
To me it looks like it is just this test which is a bit odd because we specify an ISO as `HDD_1` here. But maybe it would actually make sense to nevertheless check the extension here. Note that the further checks below actually check whether the cd overlays have the format `raw`.

Reply:
https://github.com/os-autoinst/os-autoinst/pull/1765